src/shogun/evaluation/SplittingStrategy.cpp

```diff
@@ -69,13 +69,11 @@ CSplittingStrategy::~CSplittingStrategy()
 	SG_UNREF(m_subset_indices);
 }
 
-SGVector<index_t> CSplittingStrategy::generate_subset_indices(index_t subset_idx)
+const SGVector<index_t> CSplittingStrategy::validation(index_t subset_idx) const
```

Comment:
I think it is a good idea to make this method const (if things are precomputed)
However, there is a "build_subsets" call below, which cannot be const, so the current const doenst make sense

Reply:
the function arg can be const :)